### PR TITLE
[Accounts] Update to Xcode 9 Beta 1

### PR DESCRIPTION
**Pull request by dalexsoto**
Into and xtro are happy locally 😄

### CODE REVIEW

File: src/accounts.cs [hunk 1]
```diff
@@ -112,6 +113,12 @@ interface ACAccountStore {
 		[Wrap ("RequestAccess (accountType, options == null ? null : options.Dictionary, completion)")]
 		[Async]
 		void RequestAccess (ACAccountType accountType, [NullAllowed] AccountStoreOptions options, ACRequestCompletionHandler completion);
+
+		// For some reason this was never added before, xtro caught it.
```
Comment (spouliot) [inline src/accounts.cs]: Move this to the commit message

Comment (spouliot) [inline src/accounts.cs]: We had nothing missing for iOS in xtro...
If it was in a data file it should be removed

Comment (dalexsoto) [inline src/accounts.cs]: Nope, it was inside unclassified 

`common.unclassified:!missing-selector! ACAccountStore::removeAccount:withCompletionHandler: not bound`

Comment (spouliot) [inline src/accounts.cs]: Ok, let's remove it from that file then 
And add the mac attribute

File: src/accounts.cs [hunk 2]
```diff
@@ -127,29 +133,40 @@ interface ACAccountType : NSSecureCoding {
 		[Export ("accessGranted")]
 		bool AccessGranted { get;  }
 
+		[Deprecated (PlatformName.iOS, 11, 0, message: "Use Twitter SDK instead")]
```
Comment (VincentDondain) [inline src/accounts.cs]: https://github.com/xamarin/xamarin-macios/wiki/BINDINGS#availability-attributes-messages (:

Please add a dot at the end of the sentance.

File: src/accounts.cs [hunk 3]
```diff
@@ -127,29 +133,40 @@ interface ACAccountType : NSSecureCoding {
...
+		[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use Twitter SDK instead")]
```
Comment (VincentDondain) [inline src/accounts.cs]: Same (dot).

File: src/accounts.cs [hunk 4]
```diff
@@ -127,29 +133,40 @@ interface ACAccountType : NSSecureCoding {
...
 		[Field ("ACAccountTypeIdentifierTwitter")]
 		NSString Twitter { get; }
 
+		[Deprecated (PlatformName.iOS, 11, 0, message: "Use Sina Weibo SDK instead")]
+		[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use Sina Weibo SDK instead")]
```
Comment (VincentDondain) [inline src/accounts.cs]: Same (dot).

File: src/accounts.cs [hunk 5]
```diff
@@ -127,29 +133,40 @@ interface ACAccountType : NSSecureCoding {
...
 		[Since (6,0)]
 		[Field ("ACAccountTypeIdentifierSinaWeibo")]
 		NSString SinaWeibo { get; }
 
+		[Deprecated (PlatformName.iOS, 11, 0, message: "Use Facebook SDK instead")]
+		[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use Facebook SDK instead")]
```
Comment (VincentDondain) [inline src/accounts.cs]: Same (dot).

File: src/accounts.cs [hunk 6]
```diff
@@ -127,29 +133,40 @@ interface ACAccountType : NSSecureCoding {
...
 		[Since (6,0)]
 		[Field ("ACAccountTypeIdentifierFacebook")]
 		NSString Facebook { get; }
 
+		[Deprecated (PlatformName.iOS, 11, 0, message: "Use Tencent Weibo SDK instead")]
+		[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use Tencent Weibo SDK instead")]
```
Comment (VincentDondain) [inline src/accounts.cs]: Same (dot).

File: src/accounts.cs [hunk 7]
```diff
@@ -127,29 +133,40 @@ interface ACAccountType : NSSecureCoding {
...
 		[Since (7,0)]
 		[Mac (10,9)]
 		[Field ("ACAccountTypeIdentifierTencentWeibo")]
 		NSString TencentWeibo { get; }
 
 #if MONOMAC
+		[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use LinkedIn SDK instead")]
```
Comment (VincentDondain) [inline src/accounts.cs]: Same (dot).

File: src/accounts.cs [hunk 8]
```diff
@@ -127,29 +133,40 @@ interface ACAccountType : NSSecureCoding {
...
 		[Mac (10,9)]
 		[Field ("ACAccountTypeIdentifierLinkedIn")]
 		NSString LinkedIn { get; }
 #endif
 	}
 
+	[Deprecated (PlatformName.iOS, 11, 0, message: "Use Facebook SDK instead")]
+	[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use Facebook SDK instead")]
```
Comment (VincentDondain) [inline src/accounts.cs]: Same (dot).

File: src/accounts.cs [hunk 9]
```diff
@@ -165,6 +182,8 @@ interface ACFacebookKey {
 		NSString Audience { get; }
 	}
 
+	[Deprecated (PlatformName.iOS, 11, 0, message: "Use Facebook SDK instead")]
+	[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use Facebook SDK instead")]
```
Comment (VincentDondain) [inline src/accounts.cs]: Same (dot).

File: src/accounts.cs [hunk 10]
```diff
@@ -180,6 +199,8 @@ interface ACFacebookAudienceValue
 		NSString OnlyMe { get; }
 	}
 
+	[Deprecated (PlatformName.iOS, 11, 0, message: "Use Tencent Weibo SDK instead")]
+	[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use Tencent Weibo SDK instead")]
```
Comment (VincentDondain) [inline src/accounts.cs]: Same (dot).

File: src/accounts.cs [hunk 11]
```diff
@@ -189,6 +210,7 @@ interface ACTencentWeiboKey {
 	}
 
 #if MONOMAC
+	[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use LinkedIn SDK instead")]
```
Comment (VincentDondain) [inline src/accounts.cs]: Same (dot).

**Comment by spouliot** [comment]
@vincentdondain please create an intro test to check for this (and other comments you add like quotes)

**Comment by dalexsoto** [comment]
@VincentDondain Fixed, thanks!

**Pull request merged**